content/api/commands/intercept.md

```diff
@@ -170,66 +128,60 @@ The `routeHandler` defines what will happen with a request if the [routeMatcher]
 
 ## Examples
 
-### Matching URL
+### Matching `url`
 
-You can provide the entire URL to match
+You can provide the exact [URL](#Arguments) to match, but typically pattern-matching is more effective and a glob pattern will likely be the simpler of the two to implement. See [Glob Pattern Matching URLs](#Glob-Pattern-Matching-URLs).
 
 ```js
-// will match any request that exactly matches the URL
-//   matches GET https://prod.cypress.io/users
-//   won't match GET https://staging.cypress.io/users
+// match any request that exactly matches the URL
 cy.intercept('https://prod.cypress.io/users')
-```
 
-You can provide a [`minimatch`](/api/utilities/minimatch) pattern
+// match any request that satisfies a glob pattern
+cy.intercept('/users?_limit=+(3|5)')
 
-```javascript
-// will match any HTTP method to urls that end with 3 or 5
-cy.intercept('**/users?_limit=+(3|5)')
+// match any request that satisfies a regex pattern
+cy.intercept(/\/users\?_limit=(3|5)$/)
 ```
 
-**Tip:** you can evaluate your URL using DevTools console to see if the [`minimatch` pattern](https://www.npmjs.com/package/minimatch) is correct.
+### Matching `method`
 
-```javascript
-// paste into the DevTools console while Cypress is running
-Cypress.minimatch(
-  'https://jsonplaceholder.cypress.io/users?_limit=3',
-  '**/users?_limit=+(3|5)'
-) // true
-
-// print verbose debug information
-Cypress.minimatch(
-  'https://jsonplaceholder.cypress.io/users?_limit=3',
-  '**/users?_limit=+(3|5)',
-  { debug: true }
-) // true + lots of debug messages
-```
+<Alert type="warning">
 
-You can even add an assertion to the test itself to ensure the URL is matched
+If you don't pass in a [`method` argument][arg-method], then all HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, `DELETE`, etc.) will match.
 
-```javascript
-// arguments are url and the pattern
-expect(
-  Cypress.minimatch(
-    'https://jsonplaceholder.cypress.io/users?_limit=3',
-    '**/users?_limit=+(3|5)'
-  ),
-  'Minimatch test'
-).to.be.true
+</Alert>
+
+```js
+cy.intercept('/users')
+// matches this: GET http://localhost/users
+// ...and this, too: POST http://localhost/users
+
+cy.intercept('GET', '/users')
+// matches this: GET http://localhost/users
+// ...but not this: POST http://localhost/users
 ```
 
-For the most powerful matching, provide a regular expression
+### Aliasing a Route
 
-```javascript
-cy.intercept(/\/users\?_limit=(3|5)$/).as('users')
-cy.get('#load-users').click()
-cy.wait('@users').its('response.body').should('have.length', 3)
+While `cy.intercept` doesn't yield anything, you can chain [`.as`](/api/commands/as) to it to create an [alias](/guides/core-concepts/variables-and-aliases#Aliases):
 
-// intercepts _limit=5 requests
-cy.get('#load-five-users').click()
-cy.wait('@users').its('response.body').should('have.length', 5)
+```js
+cy.intercept('GET', '/users').as('getAllUsers')
+cy.intercept('POST', '/users').as('createUser')
 ```
 
+<Alert type="info">
```

Comment:
I was hoping for this kind of feedback because I agree with you but I've been neck-deep in this for so long that it's almost impossible to have that perspective.

Reply:
Done